bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/WprimeToTauNu_LO/WprimeToTauNu_M-6000_kR-0.01_LO_NNPDF31nnlo/WprimeToTauNu_M-6000_kR-0.01_LO_NNPDF31nnlo_run_card.dat

```diff
@@ -0,0 +1,284 @@
+ False        = fixed_fac_scale  ! if .true. use fixed fac scale
+ 91.188  = scale            ! fixed ren scale
+ 91.188  = dsqrt_q2fact1    ! fixed fact scale for pdf1
+ 91.188  = dsqrt_q2fact2    ! fixed fact scale for pdf2
+ -1 = dynamical_scale_choice ! Choose one of the preselected dynamical choices
+ 1.0  = scalefact        ! scale factor for event-by-event scales
+#*********************************************************************
+# Type and output format
+#*********************************************************************
+  False     = gridpack  !True = setting up the grid pack
+  -1.0 = time_of_flight ! threshold (in mm) below which the invariant livetime is not written (-1 means not written)
+  3.0 = lhe_version       ! Change the way clustering information pass to shower.        
+  True = clusinfo         ! include clustering tag in output
+  average =  event_norm       ! average/sum. Normalization of the weight in the LHEF
+
+#*********************************************************************
+# Matching parameter (MLM only)
+#*********************************************************************
+ 0 = ickkw            ! 0 no matching, 1 MLM
+ 1.0 = alpsfact         ! scale factor for QCD emission vx
+ False = chcluster        ! cluster only according to channel diag
+ 4 = asrwgtflavor     ! highest quark flavor for a_s reweight
+ False  = auto_ptj_mjj  ! Automatic setting of ptj and mjj if xqcut >0
+                                   ! (turn off for VBF and single top processes) 
+ 0.0   = xqcut   ! minimum kt jet measure between partons
+#*********************************************************************
+#
+#*********************************************************************
+# handling of the helicities:
+#  0: sum over all helicities
+#  1: importance sampling over helicities
+#*********************************************************************
+   0  = nhel          ! using helicities importance sampling or not.
+#*********************************************************************
+# Generation bias, check the wiki page below for more information:   *
+#  'cp3.irmp.ucl.ac.be/projects/madgraph/wiki/LOEventGenerationBias' *
+#*********************************************************************
+ None = bias_module  ! Bias type of bias, [None, ptj_bias, -custom_folder-]
+ {} = bias_parameters ! Specifies the parameters of the module.
+#
+#*******************************                                                 
+# Parton level cuts definition *
+#*******************************                                     
+#                                                                    
+#
+#*********************************************************************
+# BW cutoff (M+/-bwcutoff*Gamma) ! Define on/off-shell for "$" and decay  
+#*********************************************************************
+  10000000000.  = bwcutoff      ! (M+/-bwcutoff*Gamma)
```

Comment:
Why is bwcutoff set to such a high value? @menglu21 this looks dangerous to me, can you also comment on that?

Reply:
Hi @mlizzo , 
The bwcutoff value is set to such a high number because one misses an offshell production when there are large masses and small couplings. For example, mass 5 TeV and kR 0.0001, i.e. the resulting width is very tiny. So, if I have a mass of 5 TeV and a width of 1e-6 and I want to have a distribution to 0, then I need Gamma*bwcutoff to be 5000, so 5000/1e-06 already results in a high number and I took a bit higher number just to be safe.
I checked it for cases with larger width, and this bwcutoff value seems to work fine for those cases as well.

Reply:
+ it's disapproved by the authors themselves

Reply:
@sihyunjeon Then I guess we could change it to the highest value that is possible. Which value do you recommend to use that would not go into conflict with the shower step?

Reply:
it's much smaller value 15-30. default is 15

Reply:
I do not think that the value 15 can work for my case. Could not I at least use bwcutoff = 15000? Like it is also mentioned here: 
https://twiki.cern.ch/twiki/bin/view/CMS/QuickGuideMadGraph5aMCatNLO

Reply:
i think the twiki page is saying the "equivalent" thing, because for LO process "p p > e+ vl", the only diagram is through w boson propagator, which could be onshell and offshell. so if you use syntax "p p > w+ > e+ vl", and set the bwcutoff be a large number, then the offshell region is added automatically.

Reply:
Hi as Meng said,
`generate p p > wp+ > ta+ vt` would already give offshell effect (although this syntax is also not so nice - it's better than super large bwcutoff).
You only miss the offshell contribution when you do `p p > wp+, wp+ > ta+ vt` not `p p > wp+ > ta+ vt`

Reply:
actually, since the width is so small, i don't think the bwcutoff will even have effect on the result. even though the very offshell region is now becoming onshell by using very large bwcutoff, the probability for a event having such offshell event doesn't change.

Reply:
Hi @menglu21 and @sihyunjeon, 
Thank you for your answers.
I tested the results with this syntax `generate p p > wp+ > ta+ vt ` for bwcutoff 15 and bwcutoff 15000. As I mentioned before, for large masses and small couplings, when the width is tiny, using bwcutoff of 15000 is more or less okay, whereas when I use the value of 15, I miss the offshell production.. But I guess if I do so, then it also goes into conflict with the showering step?
As I understand, there are two possibilities: 
1. Stick to the syntax that I have `generate p p > wp+ > ta+ vt ` and `generate p p > wp+ > ta- vt~` , using bwcutoff 15.
2. Switch to the syntax `p p > ta+ vt NP==2` and `p p > ta- vt~ NP==2` and use bwcutoff 15000?
Sorry for asking too many questions. I just want to figure out, which option is the best, so that I start producing cards and gridpacks with correct settings this time.

Reply:
Hi @valesark , i had some offline discussion with Sihyun, sorry i didn't make it clear, actually for your case, these two syntaxs are the same since there is only s-channel. and also value of bwcutoff won't have effect.

Reply:
Hi @menglu21,
Thank you for your explanations. 
Okay, then I will leave the syntax like it is and change the bwcutoff value to 15.

Reply:
@valesark 

If you look at https://twiki.cern.ch/twiki/bin/view/CMS/MGSyntaxPitfall, under ` Avoid using ">"s in series without ","s `
it will show you that bwcutoff 15 or 15000 will give equivalent physics (yellow black lines are identical), you would not see much difference in your cross section whether you set it to 15 or 15000 if the studies we did earlier was correct.

Can you compare cross sections between 15 and 15000?

Reply:
@sihyunjeon ,  I checked it and the difference is small, indeed. I will create new cards now.